### PR TITLE
ci: addeed github actions workflow

### PR DESCRIPTION
**Pull request by audrium**
Here is an example of GH actions workflow run: https://github.com/audrium/reana-demo-cms-h4l/runs/1346584551

Please note that we are not preserving test case described in the travis ci file. It could be extracted to `runt-tests.sh` if needed

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
```
Comment (ParthS007) [inline .github/workflows/ci.yml:21]: Minor: why not `3.8` since it is the latest?

**Review by ParthS007 — COMMENTED** [review]
LGTM

File: .github/workflows/ci.yml (hunk 1 above)
Comment (mvidalgarcia) [reply .github/workflows/ci.yml:21]: I was like this on Travis, but I agree.

Comment (audrium) [reply .github/workflows/ci.yml]: Good catch. Fixed it

**Review by mvidalgarcia — APPROVED** [review]

**Comment by mvidalgarcia** [comment]
Cannot merge due to branch protection:

```console
To github.com:reanahub/reana-demo-cms-h4l
 ! [remote rejected] master -> master (protected branch hook declined)
error: failed to push some refs to 'git@github.com:reanahub/reana-demo-cms-h4l'
```

**Pull request merged**